trustchain_design.md

```diff
@@ -112,25 +112,25 @@ The author also stores the block they just emitted, preventing the *Trustchain*
 
 ### Trust on first use
 
-When first receiving a *device* creation block for a *user*, the recipient needs to trust that the block relates to the correct *user*. The block and the information it contains &mdash; notably the new *device*'s [Device Encryption Key Pair] and [Device Signature Key Pair] &mdash; are then stored locally, preventing the need to ask for this *device*'s public key again.
+When first receiving a *device* creation block for a *user*, the recipient needs to trust this block relates to the correct *user*. The block and the information it contains &mdash; especially the new *device*'s [Device Encryption Key Pair] and [Device Signature Key Pair] &mdash; are then stored locally, preventing the need to ask for this *device*'s public key again.
 
-It is to be noted that because of the signature delegation scheme, the entity that needs to be trusted upon receiving blocks for a new *user* is **not** the *Trustchain* distribution system, but the private [Trustchain Signature Key Pair] holder (i.e. the *application server*).
+It has to be noted that because of the signature delegation scheme, the entity that needs to be trusted upon receiving blocks for a new *user* is **not** the *Trustchain* distribution system, but the private [Trustchain Signature Key Pair] holder (i.e. the *application server*).
 
 ### Forward secrecy
 
-Forward secrecy is, despite being an interesting property, **not **implemented in *Tanker Core*. The reason is that *Tanker Core* is not targeted solely at encrypting ephemeral message exchanges, but also persistent data. We want *users* to be able to decrypt anything at any time with any of their *devices*, which means that we cannot use a forward secure scheme.
+Forward secrecy is, despite being an interesting property, **not** implemented in *Tanker Core*. The reason is that *Tanker Core* is not targeted solely at encrypting ephemeral message exchanges, but also persistent data. We want *users* to be able to decrypt anything at any time with any of their *devices*, which means that we cannot use a forward secure scheme.
 
-We consider implementing an optional forward secure scheme tailored for messaging in a future version of *Tanker Core*.
+We are considering implementing an optional forward secure scheme tailored for messaging in a future version of *Tanker Core*.
```

Comment:
delete this? this is not an advertisement flyer

Reply:
yes